spec/factories/plate_factories.rb

```diff
@@ -84,9 +126,14 @@
         evaluator.ancestors.select { |a| parameters[:purpose_name].include?(a.purpose.name) }
       end
       RSpec::Mocks.allow_message(plate, :ancestors).and_return(ancestors_scope)
-      RSpec::Mocks.allow_message(plate, :parents).and_return(evaluator.parents)
+      plate._cached_relationship(:parents) { evaluator.parents }
     end
 
+    # Set up a stock plate. Changes:
```

Comment:
When it says 'changes', what does it mean? Changes with respect to what? Cheers

Reply:
Relative to the parent factory for a standard plate. Updated the comment.